website/docs/index.html.markdown

```diff
@@ -373,6 +373,102 @@ The following arguments are used to configure the VMware Cloud Director Provider
 * `import_separator` - (Optional; *v2.5+*) The string to be used as separator with `terraform import`. By default
   it is a dot (`.`).
 
+* `ignore_metadata_changes` - (Optional; *v3.10+*) Use one or more of these blocks to ignore specific metadata entries from being changed by this Terraform provider
+  after creation or when they were created outside Terraform.
+  See ["Ignore Metadata Changes"](#ignore-metadata-changes) for more details.
+
+## Ignore metadata changes
+
+One or more `ignore_metadata_changes` blocks can be optionally set in the provider configuration, which will allow to ignore specific `metadata_entry`
+items during `plan`, `updates-in-place` and `destroy` (**not** `create`). This is useful, for example, to avoid removing metadata entries that were created
```

Comment:
How can this affect `destroy`?
Doesn't `destroy` affect the whole resource, regardless of its contents?

Reply:
This is wrong indeed. I've changed the whole sentence to avoid specifying the operations.